app/Presenters/Admin/ArtworkPresenter.php

```diff
@@ -351,7 +351,15 @@ protected function getArtworkDetailsBlock()
         } else {
             if (!empty($this->entity->date_block)) {
                 $details[] = [
-                    'key' => 'Date',
+                    'key' => '<span id="h-date-disclaimer">Date</span>&nbsp;'
+                    . '<button class="info-button-trigger" data-behavior="infoButtonTrigger" aria-label="Info" aria-expanded="false" data-breakpoints="none">'
+                    . '    <svg class="icon--info"><use xlink:href="#icon--info" /></svg>'
+                    . '</button>'
+                    . '<span class="info-button-info s-hidden" id="info-button-info-date-disclaimer" data-behavior="infoButtonInfo">'
+                    . '    <span class="f-caption">'
+                    . '        Dates are not always precisely known, but the Art Institute strives to present this information as consistently and legibly as possible. Dates may be represented as a range that spans decades, centuries, dynasties, or periods and may include qualifiers such as c. (circa) or BCE.'
+                    . '    </span>'
+                    . '</span>',
```

Comment:
This is perfectly fine and consistent with the reset of the file, but for long strings, I like to use [heredocs](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc), for example:
```php
'key' => <<<VALUE
<span id="h-date-disclaimer">Date</span>&nbsp;
<button class="info-button-trigger" data-behavior="infoButtonTrigger" aria-label="Info" aria-expanded="false" data-breakpoints="none">
  <svg class="icon--info"><use xlink:href="#icon--info" /></svg>
</button>
<span class="info-button-info s-hidden" id="info-button-info-date-disclaimer" data-behavior="infoButtonInfo">
  <span class="f-caption">
      Dates are not always precisely known, but the Art Institute strives to present this information as consistently and legibly as possible. Dates may be represented as a range that spans decades, centuries, dynasties, or periods and may include qualifiers such as c. (circa) or BCE.
  </span>
</span>
VALUE;
```